app/src/editor/canvas/components/Ports/Value/Select/select.pcss

```diff
@@ -19,8 +19,7 @@
   outline: 0;
   padding: 0 calc(0.5 * var(--um));
   transition: 200ms linear;
-  transition-property: background-color, color, width, box-shadow;
-  width: 100%;
+  transition-property: background-color, color, box-shadow;
```

Comment:
ah. The solution was no width at all. 👏 👏

Reply:
Yep, you don't wanna `width: 100%` on the invisible `select`.